src/video/renderers/null/null_renderer.c

```diff
@@ -0,0 +1,98 @@
+#include "video/renderers/null/null_renderer.h"
+
+#include "utils/allocator.h"
+#include "utils/log.h"
+#include "video/vga_state.h"
+
+static bool is_available(void) {
+    return true;
+}
+
+static const char *get_description(void) {
+    return "Zero output renderer";
+}
+
+static const char *get_name(void) {
+    return "NULL";
+}
+
+static bool setup_context(void *userdata, int window_w, int window_h, bool fullscreen, bool vsync) {
+    INFO("NULL Renderer initialized!");
+    return true;
+}
+
+static void get_context_state(void *userdata, int *window_w, int *window_h, bool *fullscreen, bool *vsync) {
+    if(window_w != NULL)
+        *window_w = 0;
+    if(window_h != NULL)
+        *window_h = 0;
```

Comment:
duplicate lines here

Reply:
no?
window_w and window_h are different pointers

Reply:
yeah, I saw wrong.